setup.py

```diff
@@ -88,5 +88,6 @@ def readme():
         'Topic :: Software Development :: Libraries :: Application Frameworks'
     ],
     include_package_data=True,  # needed to include swagger-ui (see MANIFEST.in)
-
+    entry_points={'console_scripts': ['connexion = connexion.cli:main',
+                                      'cnx = connexion.cli:main']}
```

Comment:
as already mentioned: I would remove the `cnx` alias --- I think it adds no value here (users can always create as many aliases as they wish via their CLI...)